build/cmake/find/FindGMock.cmake

```diff
@@ -60,71 +60,70 @@
 
 
 function(_gmock_append_debugs _endvar _library)
-  if(${_library} AND ${_library}_DEBUG)
-    set(_output optimized ${${_library}} debug ${${_library}_DEBUG})
-  else()
-    set(_output ${${_library}})
-  endif()
-  set(${_endvar} ${_output} PARENT_SCOPE)
+	if(${_library} AND ${_library}_DEBUG)
+		set(_output optimized ${${_library}} debug ${${_library}_DEBUG})
+	else()
+		set(_output ${${_library}})
+	endif()
+	set(${_endvar} ${_output} PARENT_SCOPE)
 endfunction()
 
 function(_gmock_find_library _name)
-  find_library(${_name}
-    NAMES ${ARGN}
-    HINTS
-      $ENV{GMOCK_ROOT}
-      ${GMOCK_ROOT}
-    PATH_SUFFIXES ${_gmock_libpath_suffixes}
-  )
-  mark_as_advanced(${_name})
+	find_library(${_name}
+		NAMES ${ARGN}
+		HINTS
+			$ENV{GMOCK_ROOT}
+			${GMOCK_ROOT}
+		PATH_SUFFIXES ${_gmock_libpath_suffixes}
+	)
+	mark_as_advanced(${_name})
 endfunction()
 
 
 if(NOT DEFINED GMOCK_MSVC_SEARCH)
-  set(GMOCK_MSVC_SEARCH MD)
+	set(GMOCK_MSVC_SEARCH MD)
 endif()
 
 set(_gmock_libpath_suffixes lib)
 if(MSVC)
-  if(GMOCK_MSVC_SEARCH STREQUAL "MD")
-    list(APPEND _gmock_libpath_suffixes
-      msvc/gmock-md/Debug
-      msvc/gmock-md/Release)
-  elseif(GMOCK_MSVC_SEARCH STREQUAL "MT")
-    list(APPEND _gmock_libpath_suffixes
-      msvc/gmock/Debug
-      msvc/gmock/Release)
-  endif()
+	if(GMOCK_MSVC_SEARCH STREQUAL "MD")
+	list(APPEND _gmock_libpath_suffixes
+		msvc/gmock-md/Debug
+		msvc/gmock-md/Release)
+	elseif(GMOCK_MSVC_SEARCH STREQUAL "MT")
+	list(APPEND _gmock_libpath_suffixes
+		msvc/gmock/Debug
+		msvc/gmock/Release)
+	endif()
 endif()
 
 find_path(GMOCK_INCLUDE_DIR gmock/gmock.h
-  HINTS
-    $ENV{GMOCK_ROOT}/include
-    ${GMOCK_ROOT}/include
-)
+		HINTS
+		$ENV{GMOCK_ROOT}/include
+		${GMOCK_ROOT}/include)
+
 mark_as_advanced(GMOCK_INCLUDE_DIR)
 
 if(MSVC AND GMOCK_MSVC_SEARCH STREQUAL "MD")
-  # The provided /MD project files for Google Mock add -md suffixes to the
-  # library names.
-  _gmock_find_library(GMOCK_LIBRARY            gmock-md  gmock)
-  _gmock_find_library(GMOCK_LIBRARY_DEBUG      gmock-mdd gmockd)
-  _gmock_find_library(GMOCK_MAIN_LIBRARY       gmock_main-md  gmock_main)
-  _gmock_find_library(GMOCK_MAIN_LIBRARY_DEBUG gmock_main-mdd gmock_maind)
+	# The provided /MD project files for Google Mock add -md suffixes to the
+	# library names.
+	_gmock_find_library(GMOCK_LIBRARY				gmock-md  gmock)
```

Comment:
The column with the items gmock-md... here should be aligned in the same way it was before.